src/main/ccl/cclutassert_declare.inc

```diff
@@ -6,6 +6,8 @@
 ;
 ;**********************************************************************************************************************************
 
+declare cclut::ignoreCclStringLogic = i2 with protect, noconstant(FALSE)
```

Comment:
"ignore CCL string logic" is not the best name for this.
When it is TRUE, CCL string logic \***is\*** used.